Firmware/ControladorRobo/ControladorRobo.ino

```diff
@@ -46,8 +47,6 @@ void setup()
 
     HBridge::self();
```

Comment:
Because the encoder.h no longer exists, and I didn't think there was enough content to justify it's existence.
Suggestions?